lib/app/sequences/04_provider_ehr_launch_sequence.rb

```diff
@@ -0,0 +1,257 @@
+module Inferno
+  module Sequence
+    class ProviderEHRLaunchSequence < SequenceBase
```

Comment:
Remove 'Provider'?

Reply:
I'll do this after.  This needs more testing and isn't visible on the web interface.